### ISSUE TITLE
Gym?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#114]
Health and fitness are core goals of @home.

Some people find it easier to exercise at a dedicated place like a gym than to exercise at home.
I'm a fan of doing [bodyweight exercises](https://github.com/nelsonic/nelsonic.github.io/blob/master/p/goals.md#13-exercise-every-morning) @home and have been doing them every day for the past couple of years. But I know this is not for everyone and I _definitely_ prefer going to a _class_ with a coach.

There are _many_ gyms in Braga https://www.google.com/search?tbm=lcl&q=gym+braga 🏋 
![image](https://user-images.githubusercontent.com/194400/73465599-81245580-4378-11ea-99a9-a9a9d2ffd328.png)

I've done quite a lot of research into this and only a handful allow you to join without a contract. 💰 
> Some gyms require a **joining fee** and a **12 month contract**. This is utterly absurd, but I'm guessing they are well-established, have good locations and don't want to let any undesirables join so they have a ridiculous barrier to entry. An _even_ more cynical perspective on this is that I did most of my enquiries during January (_the traditional ["new year's resolution"](https://youtu.be/Pm9CQn07OjU) month_) 

FlexGym is the nearest gym to @home that has no joining fee or contract: [flexgym.pt](http://www.flexgym.pt/modalidades)
The site does not have an english version so Google Translate is your friend if you need it.
![image](https://user-images.githubusercontent.com/194400/73465111-cbf19d80-4377-11ea-9fb5-b631ba430b43.png)

Price plans: http://www.flexgym.pt/precoseplanos
![image](https://user-images.githubusercontent.com/194400/73468052-1ecd5400-437c-11ea-9ca9-0874dfc24623.png)
+ €24.90/month if you only want to use the gym and one type of group exercise e.g "Cross Fit".
+ €29.90/month includes a martial art class e.g: Karate, JiuJitsu, MMA, Boxing etc. 
but you can only **pick one** of these classes to attend. You can switch once per month. 
(_I asked if I could pay extra to sample all of the martial arts in a single month and got a "no"..._)
+ €36.90/month includes _everything_ and you can pick **_two_ types** of martial arts. 

I am thinking of _testing_ FlexGym for the month of February 2002. 
(_yes, I know it's the shortest i.e. most expensive month of the year!_ 🙄) 

+ [ ] 🧘Pilates: http://www.flexgym.pt/mapa-aulas/#pilates > Tues + Thur 20:00 - 20:45
+ [ ] 🥋Karate: http://www.flexgym.pt/mapa-aulas/#karate-adulto > Mon + Wed 20:00 - 21:00
+ [ ] 💪 [Strength training](https://www.health.harvard.edu/staying-healthy/want-to-live-longer-and-better-strength-train) (_lifting weights in the gym_) > Mon - Fri 07:30 - 22:00

@iteles / @edwardcodes are you interested in joining for either of these classes?
If we _all_ join, then we get the "**_everything_ included**" membership for €29.95 (each)
![image](https://user-images.githubusercontent.com/194400/73468951-820bb600-437d-11ea-8620-239d02ae22c4.png)

I'm happy to be a "test dummy" on this and report back if you don't have time right now. 👍 

> **Note**: We are _considering_ building a multi-purpose exercise room in the [_basement_](https://github.com/dwyl/home/issues/14#issuecomment-504559126) of the Sun Room.
It's only 4m x 8m and would never be as good as a "real" Gym ... 
However we think it's perfect for a "yoga studio" where people staying @home can congregate and follow a _free_ training video. e:g https://youtu.be/v7AYKMP6rOE
[![image](https://user-images.githubusercontent.com/194400/73481541-ee91af80-4393-11ea-9acb-88a16c08314b.png)](https://youtu.be/v7AYKMP6rOE)
But the Sun Room will not be built for a few more months, so I think it's worth testing FlexGym

> There is also _free_ calisthenics park a short walk from @home:
https://calisthenics-parks.com/spots/6724-en-sao-victor-calisthenics-park-r-da-fabrica
![image](https://user-images.githubusercontent.com/194400/73481133-12082a80-4393-11ea-874a-b0babd748ce2.png)
But from experience, the "short walk" becomes _very_ long when it's raining ... 💭

**Comment by iteles** [comment]
I'm interested in trialling the classes before I join. I'm on the back foot about it not even mentioning the style of karate on the website...!

**Comment by nelsonic** [comment]
Yeah, it's _unclear_ from the website what `style` of Karate it is.
I've never had any issue with Shotokan, it's the _teachers_ who sometimes spoil it with over-emphasis on Kumite ... Shotokan is _meant_ to practice the Standard JKA Katas [wikipedia.org/wiki/Shotokan#Kata](https://en.wikipedia.org/wiki/Shotokan#Kata)

<hr />

A bit of desk detective work ...
The PT Karate Federation http://fnkp.pt lists all the clubs/dojos that are members in their annual list:
http://fnkp.pt/wp-content/uploads/2019/07/Lista_Clubes_2018_9Jul19.pdf
FlexGym is listed as being a member of **AWIKP** - Associação Wado Internacional Karate-Do Portugal
![image](https://user-images.githubusercontent.com/194400/73490327-299bdf00-43a4-11ea-95eb-193143fcf205.png)
http://wadoryu-portugal.com
![image](https://user-images.githubusercontent.com/194400/73490868-4c7ac300-43a5-11ea-914a-ddbc1ebad59c.png)

https://en.wikipedia.org/wiki/World_Karate_Federation
![image](https://user-images.githubusercontent.com/194400/73491099-c90da180-43a5-11ea-8fb1-311b8d920d57.png)

https://en.wikipedia.org/wiki/Wad%C5%8D-ry%C5%AB
Wadō-ryū is the only style of Karate recognised by WKF that I have _not_ practiced. 
**Wa** means "***harmony***," **dō** (same character as tao) means "**way**," and **ryū** means "**school**".
"**_Harmony_ Way School**" doesn't sound like it would incur too many broken bones during practice. 😉 

So I'm keen to give it a `try`. 
We would need start at white belt again, but I'm cool with that. 
[`#shoshin`](https://en.wikipedia.org/wiki/Shoshin)
<hr />

Even if you don't do/like the Karate, would you pay €30/month for 8 pilates lessons? (€3.75 each)

**Comment by iteles** [comment]
I would, if I liked the class. But also, who wants to go to the gym at 20h00 at night? 🙄 Id like to try it out!

**Comment by edwardcodes** [comment]
@nelsonic Off course!! Gym and Karate sounds amazing 😉

**Comment by nelsonic** [comment]
@edwardcodes do you have a Kimono ("Gui") for Karate? (_if you still have a JiuJitsu one it works_) 🥋
If not, we will visit Decathlon and buy one for you. 👍 
Bring your gym clothes and travel towel with you. 🚿 
Also, don't discount Pilates for core strength and toning; it's a good precursor to calisthenics ... 💭

**Comment by iteles** [comment]
We've been going to karate for the past month and it's excellent ❤️
